composer.json

```diff
@@ -59,7 +59,11 @@
         "symfony/http-kernel": "^3.0",
         "symfony/intl": "^3.0",
         "symfony/options-resolver": "^3.0",
-        "symfony/polyfill": "^1.6",
+        "symfony/polyfill-apcu": "~1.1",
+        "symfony/polyfill-intl-icu": "~1.0",
+        "symfony/polyfill-mbstring": "~1.0",
+        "symfony/polyfill-php56": "~1.0",
```

Comment:
is this needed when we support only 7.1? /cc @wachterjohannes

Reply:
i think the only thing which is realy needed is `symfony/polyfill-mbstring` and `symfony/polyfill-apcu` (but only in `sulu-minimal` and `sulu-standard`

Reply:
@wimvds what would you like to achieve with that? had you any problem without that?